web/app/view/dialog/ReportConfigController.js

```diff
@@ -27,22 +27,23 @@ Ext.define('Traccar.view.dialog.ReportConfigController', {
 
     onSaveClick: function (button) {
         var eventType;
-        this.getView().callingPanel.deviceId = this.lookupReference('deviceField').getValue();
-        this.getView().callingPanel.groupId = this.lookupReference('groupField').getValue();
+        var callingPanel = this.getView().callingPanel;
```

Comment:
Please define all variable in the first line. If assignment is too long, split definition and assignment.